.env.example

```diff
@@ -27,6 +27,11 @@ MAILGUN_SMTP_SERVER=
 NEWS_TAGGER_ENDPOINT=
 NEWS_TAGGER_BASIC_AUTH_USERNAME=
 NEWS_TAGGER_BASIC_AUTH_PASSWORD=
+RDS_NAME=
```

Comment:
Let's change the ENV var names to CALENDAR_DB_XXX since we'll almost assuredly have multiple RDS databases connected in the near future.